src/index.html

```diff
@@ -1,17 +1,131 @@
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Forms</title>
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <section>
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="form-field">
+            Surname:
+            <input type="text" name="surname"> <br>
+          </label>
+          <label class="form-field">
+            Name:
+            <input type="text" name="name"><br>
+          </label>
+          <label for="quantity" class="form-field">
+            How old are You?:
+            <input
+              id="quantity"
+              type="number"
+              name="quantity"
+              placeholder="12"
+              min="1"
+              max="100"
+            ><br>
+          </label>
+          <label class="form-field">
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+              max="2001-03-20"
+              min="1939-08-19"
+            ><br >
+          </label>
+          <label class="form-field">
+            <label>I accept the term of the agreement</label>
+            <input type="checkbox" name="terms"
+            required
+            >
+          </label>
+        </fieldset>
+      </section>
+
+      <section>
+        <fieldset>
+          <legend>Registration</legend>
+          <label for="e-mail">E-mail:</label><br>
+          <input
+            placeholder="email@example.com"
+            type="text"
+            id="e-mail"
+            name="e-mail"
+          ><br>
+          <label for="pwd">Password:</label><br>
+          <input type="password" id="pwd"
+           name="pwd"
+          ><br><br>
+        </fieldset>
+      </section>
+      <section>
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+
+          Do you love cats?
+          <input type="radio" id="yes"
+          name="preferences" value="Yes"
+          >
+          <label for="yes">Yes</label>
+          <input type="radio" id="no"
+          name="preferences" value="No"
+          >
+          <label for="no">No</label> <br>
+
+          <label for="favcolor">What is your favourite colour?</label>
+          <input type="color" id="favcolor"
+          name="favcolor" value="#000000"
+          >
```

Comment:
```suggestion
          <input
            type="color" 
            id="favcolor"
            name="favcolor"
            value="#000000"
          >
```
fix code style all over the page according to the checklist